src/components/ClusterTemplateWizard/wizardValidationSchema.ts

```diff
@@ -20,7 +20,7 @@ const useWizardValidationSchema = (isCreateFlow: boolean) => {
   //It's used for testing unique names, if it fails or not loaded yet, the backend will block the creation
   const [clusterTemplates] = useClusterTemplates();
   const usedTemplateNames = React.useMemo(
-    () => clusterTemplates.map((template) => template.metadata?.name),
+    () => clusterTemplates.map((template) => template.metadata?.name).filter(Boolean) as string[],
```

Comment:
same here, `reduce` instead of `Boolean`

Reply:
Done